eras/babbage/test-suite/test/Tests.hs

```diff
@@ -25,7 +25,6 @@ defaultTests =
     [ Golden.tests
     , Tripping.tests
     , txInfoTests (Proxy @Babbage)
-    , txInfoTestsBabbageOnly
```

Comment:
That is because the only thing it was testing was presence of incompatible Plutus versions within a translation, which is no longer possible thanks to the type checker. 

https://github.com/input-output-hk/cardano-ledger/blob/bcf6051f4a0b69c82e4de7327d7d8801fa1bad87/eras/babbage/test-suite/src/Test/Cardano/Ledger/Babbage/TxInfo.hs#L346C1-L351